assets/css/stripe-styles.scss

```diff
@@ -167,6 +168,35 @@
 	position: relative;
 }
 
+.woocommerce-checkout {
+	#payment {
+		ul.payment_methods {
+			li {
+				.card-brand-icons {
+					display: block;
+					float: right;
+					width: 40px;
+					height: 24px;
+
+					img {
+						position: absolute;
+						opacity: 0;
+						transition: .3s;
+					}
+
+					img.icon-visible {
+						opacity: 1;
+					}
+				}
+
+				img {
+					float: right;
+				}
```

Comment:
Storefront forces this, but other themes don't, so the icon looks kinda weird right next to the payment method name.

By adding this we make sure the icons are displayed like in Storefront, but it might not be our responsibility to do that, so let me know what you think.